src/commcare_cloud/ansible/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -73,6 +73,12 @@ effective_cache_size = {{ postgresql_effective_cache_size }}
 enable_nestloop = {{ enable_nestloop }}
 {% endif %}
 
+# PARALLEL QUERY
+{% if postgresql_max_parallel_degree > 0 %}
+max_parallel_degree = {{ postgresql_max_parallel_degree }}
+max_worker_processes =  {{ postgresql_max_worker_processes }}
```

Comment:
should these have defaults as well?

Reply:
It will only be set when `postgresql_max_parallel_degree` is more than 0 and in `defaults.yml` it is 0

Reply:
Correct. So by default if someone sets `postgresql_max_parallel_degree = 1`, but didn't explicitly set `postgresql_max_worker_processes = x` wouldn't that  result in an invalid postgres config?

Reply:
Ohh right! Thanks, I Will fix that.